domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/Employment.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.hmcts.cmc.domain.models.statementofmeans;
+
+import lombok.Builder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import uk.gov.hmcts.cmc.domain.constraints.EachNotNull;
+import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Builder
+public class Employment {
```

Comment:
Maybe something like:

```
{ // can be either (employment and / or selfEmployment) or unemployment
  employment: { // nullable
    employers: []
  },
  selfEmployment: { // nullable
    jobTitle: "",
    annualTurnover: 0,
    onTaxPayments: { // nullable
      amountYouOwe: 0,
      reason: ""
    }
  },
  unemployment: { // nullable
    unemployed: false,
    retired: false,
    other: ""
  }
}
```

I designed this structure with validation rules in mind.

Reply:
`unemployment` also have `noOfYears` and `noOfMonths` if use chooses `unemployed`

Reply:
I was just going to make the same suggest but you guys were ahead.